vkconfig_core/layers/layers_schema.json

```diff
@@ -11,11 +11,11 @@
     "properties": {
         "file_format_version": {
             "type": "string",
-            "pattern": "^1.[0-9]+.[0-9]+"
+            "pattern": "^1.[0-9]+.[0-9]+$"
```

Comment:
I am using "$ref": "#/definitions/version", instead now so major version may change. 

I think my mind frame what this "I'd be fine with it not, assuming a new major version == new schema." but really it doesn't matter as for a new version it will be a new schema anyway so we would change the schema definition.